PendingReleaseNotes.md

```diff
@@ -15,6 +15,7 @@ v1.6...
   See https://github.com/rook/rook/issues/7275 for more information.
 
 ## Features
+* Support running Volume Replication Controller on the RBD provisioner pod.
```

Comment:
Let's move this to the ceph section below

Reply:
Does the replication controller require the CSI driver 3.3?

Reply:
Yes, should we mention that here as well?

Reply:
Yes. For we can document it. Maybe I can send a patch later to do this validation.